omod/src/main/resources/messages.properties

```diff
@@ -0,0 +1,14 @@
+#
```

Comment:
Please use the existing `messages.properties` files in the API rather than adding them to the OMOD.

Reply:
@ibacher I tried using the one in API, but it doesn't seem to work. I mean the `MessageResolver` is not fetching messages from those files.

Reply:
That's problematic. Does it work if you run this code in an actual instance of OpenMRS?

Reply:
@ibacher I'm not sure about it. I did however run this on the OpenMRS server, but I'm not sure how to check for the narrative. There is nothing for the narrative on the front-end side right? 

On the other hand, it might be that it is happening in the tests since we are using `ReloadableResourceBundleMessageSource` instead of the one which gets autowired. So for this, will setting a different `ResourceLoader` for this message source work? I'm not sure how to set it up to use the `API` files though.

Also, will keeping the messages in both the `messages.properties` be a good idea?

Reply:
@ibacher Also, aren't the `messages.properties` files supposed to be in the `/omod/src/main/resources` directory as mentioned in the [wiki](https://wiki.openmrs.org/display/docs/Module+messages.properties+Files)? Or is it configured differently in FHIR?